src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -327,11 +325,9 @@ private _SqlMetaDataSet(_SqlMetaDataSet original)
             id = original.id;
             _hiddenColumnCount = original._hiddenColumnCount;
             _visibleColumnMap = original._visibleColumnMap;
-#if !NETFRAMEWORK
             dbColumnSchema = original.dbColumnSchema;
-#else
             schemaTable = original.schemaTable;
```

Comment:
Not 100% sure about this one. Was the intention here to exclude schemaTable for netcore?

Reply:
Yeah, this kind of code doesn't make a ton of sense to me (the original, that is). It'd be nice if it came with a comment that explains what's the rationale for whats going on but ... whatever. 

As for your change, I'm also not 100% about it either. It basically says we assign the dbColumnSchema in netcore, and assign the schemaTable in netfx. So, I wonder if assigning both in both cases is the right behavior...

Reply:
I think I'm leaving this in for now